features/upgrade/api_auth/upgrade.feature

```diff
@@ -270,7 +270,6 @@ Feature: apiserver and auth related upgrade check
   @upgrade
   @network-ovnkubernetes @network-openshiftsdn
   @proxy @noproxy @disconnected @connected
-  @hypershift-hosted
```

Comment:
Hmm, sorry, I did not look carefully. After you tag me in this PR , I found it does be our case issue, its `I store the ready and schedulable masters in the clipboard` uses master but HyperShift guest has no master. Code should change to use worker. For line 293 and line 325, could you BTW help update below code? Then need not `@hypershift-hosted` can be preserved. Thank you!
```
When I store the ready and schedulable masters in the clipboard
```
Update to:
```
When I store the ready and schedulable workers in the clipboard
```

Reply:
ah, good.  I updated. PTAL..

Reply:
/lgtm
/cc @liangxia @pruan-rht